lib/ansible/modules/cloud/lxd/lxd_container.py

```diff
@@ -381,6 +388,8 @@ def _unfreeze_container(self):
         self.actions.append('unfreez')
 
     def _container_ipv4_addresses(self, ignore_devices=['lo']):
+        if self.wait_for_ipv4_addresses_ignore_devices:
+            ignore_devices = ['lo'] + self.wait_for_ipv4_addresses_ignore_devices
```

Comment:
I think it's better to use just `self.wait_for_ipv4_addresses_ignore_devices` instead of
`['lo'] + self.wait_for_ipv4_addresses_ignore_devices`,
and make the default value of `wait_for_ipv4_addresses_ignore_devices` to be `['lo']`.

In this way, the name `wait_for_ipv4_addresses_ignore_devices` respresents
actual behavior more than your current way.